app/src/main/java/com/oracle/javafx/scenebuilder/app/preferences/PreferencesWindowController.java

```diff
@@ -259,6 +265,15 @@ protected void controllerDidLoadFxml() {
         // Wildcard Imports
         wildcardImports.setSelected(recordGlobal.isWildcardImports());
         wildcardImports.selectedProperty().addListener(new WildcardImportListener());
+        
+        if (EditorPlatform.IS_MAC) {
+            // Alternate paste behavior for Text Input Controls
+            alternatePasteBehavior.setSelected(recordGlobal.isAlternateTextInputControlPaste());
+            alternatePasteBehavior.selectedProperty().addListener(new AlternatePasteListener());
+        } else {
+            alternatePasteBehavior.setDisable(true);
+            alternatePasteBehaviorLabel.setDisable(true);
```

Comment:
I prefer not showing it on other platforms. Popup doesn't show up on disabled nodes and this could confuse users on other platforms.

Reply:
Okay, agree for that. Will make this invisible for non-Mac platforms.

Reply:
The option is now only visible on Mac.